### PR TITLE
KON-316 Add LibrarySnippets

### PR DESCRIPTION
**Pull request by nataliapeterwas**

### CODE REVIEW

**Comment by nataliapeterwas** [comment]
Merge after https://github.com/LemonAppDev/konsist/pull/365

File: lib/src/snippet/kotlin/com/lemonappdev/konsist/LibrarySnippets.kt [hunk 1]
```diff
@@ -12,6 +16,35 @@ class LibrarySnippets {
...
+            .functions(includeNested = true)
+            .assert { it.hasValidParameterKDoc() }
+    }
+
+    fun `every function with return value has a return tag`() {
+        Konsist.scopeFromPackage("..api..")
+            .functions(includeNested = true)
+            .assert { it.hasValidReturnTypeKDoc() }
+    }
+
```
Comment (nataliapeterwas) [inline lib/src/snippet/kotlin/com/lemonappdev/konsist/LibrarySnippets.kt:30]: I added two options of `every extension has a receiver tag` test. I don't know which version is better.
Is any better way to filter `declarations`  by several providers (in this case we need all declarations which are KoReceiverTypeProvider and KoKDocProvider instances)?

Comment (igorwojda) [reply lib/src/snippet/kotlin/com/lemonappdev/konsist/LibrarySnippets.kt:31]: Let's leave only option 2

**Comment by nataliapeterwas** [comment]
Merge after https://github.com/LemonAppDev/konsist/pull/395

**Pull request merged**